python_scripts/ensemble_gradient_boosting.py

```diff
@@ -239,7 +239,7 @@ def generate_data(n_samples=50):
 print(f"Error of the tree: {target_true - y_pred_first_and_second_tree:.3f}")
 
 # %% [markdown]
-# We chose a sample for which only two trees were enough to make the perfect
+# We choose a sample for which only two trees were enough to make the perfect
```

Comment:
I think this one should actually be in past tense, as it refers to the sample we chose above.

Reply:
fair enough